### ISSUE TITLE
[CoE Starter Kit - QUESTION] QUESTION

### ISSUE THREAD

**Issue by EMKody137** [microsoft/coe-starter-kit#5559]
### Does this question already exist in our backlog?

- [X] I have checked and confirm this is a new question.

### What is your question?

Hello, I am currently working on setting up a COE environment and am currently getting an error when I try to upload the 'Center of Excellence Audit Components'. It produces an error that says the solution cannot be imported because I do not have the corresponding dependency 'Center of Excellence Governance Components'. The problem is in the CoE Starter kit, there is not an included solution for governance components, so my question is how to get those. The instructions for importing the governance solution just says to import the audit components, but as stated above I am unable to do so. Thanks in advance for any help!

### What solution are you experiencing the issue with?

Audit Log

### What solution version are you using?

3.8.1

### What method are you using to get inventory and telemetry?

Cloud flows

**Comment by EMKody137** [comment]
Hi Manuela,

With your guidance, here is what I found out. I had gone through the process of creating the dummy approval, however upon a second check, I was not seeing the flow approval solutions that it references in the documentation. I went into Power Automate and ran the flow again, and this time I was able to see those solution items. With those in the list I attempted again to upload the audit components, and this time it went through without error. I assume I just had ran the flow incorrectly the first time which was causing the error. I appreciate the quick response/assistance!
